common/abstracts/_variables.scss

```
@@ -28,3 +28,7 @@ $toggle-slider-color: white;
 $toggle-focus-color: black;
 
 $topic-card-big-width: 18.75rem;
+
+$default-z-index: 1;
+$fav-language-selector-z-index: calc($default-z-index + 1);
+$outline-z-index: calc($default-z-index + 1);
```

SCSS variables are static and don't need runtime calculations

```suggestion
$fav-language-selector-z-index: $default-z-index + 1;
$outline-z-index: $default-z-index + 1;
```